mlos_bench/mlos_bench/optimizers/convert_configspace.py

```diff
@@ -77,6 +77,10 @@ def _tunable_to_configspace(
     meta: Dict[Hashable, TunableValue] = {"cost": cost}
     if group_name is not None:
         meta["group"] = group_name
+    if tunable.is_numerical and tunable.quantization_bins:
+        # Temporary workaround to dropped quantization support in ConfigSpace 1.0
+        # See Also: https://github.com/automl/ConfigSpace/issues/390
+        meta[QUANTIZATION_BINS_META_KEY] = tunable.quantization_bins
```

Comment:
From here, `mlos_core.Optimizer` should be able to apply the `monkey_patch_cs_quantization` in the base class and `mlos_bench` basically shouldn't need to be aware of it.

Reply:
As discussed in https://github.com/bpkroth/MLOS/pull/60, this isn't actually true.
Not all underlying optimizers actually use ConfigSpace🤦, so we'd have to quantize at the Tunables wrapper layer and that might interact poorly with the underlying optimizers if we end up not actually registering the originally suggested value with it, but maybe that's something we can experiment with and address in the future as well.